database/factories/BeatmapsetFactory.php

```diff
@@ -10,16 +10,21 @@
 use App\Models\Beatmapset;
 use App\Models\Genre;
 use App\Models\Language;
+use App\Models\User;
 use Illuminate\Database\Eloquent\Factories\Factory;
 
 class BeatmapsetFactory extends Factory
 {
     protected $model = Beatmapset::class;
 
+    public function configure()
+    {
+        return $this->afterMaking(fn (Beatmapset $beatmapset) => $beatmapset->creator = $beatmapset->user?->username ?? $this->faker->userName());
```

Comment:
I think setting with `??=` is more appropriate here

(also the line is getting kinda long)